spec/setting_spec.rb

```diff
@@ -2,74 +2,79 @@
 
 describe ActiveSetting::Setting, 'with types/casting' do
   it 'should handle basic types and casting' do
-    s = ActiveSetting::Setting.new(:data_type => :integer, :raw_value => '6')
-    s.value.should == 6
+    s = ActiveSetting::Setting.new(data_type: :integer, raw_value: '6')
+    s.value.should eq 6
 
-    s = ActiveSetting::Setting.new(:data_type => :decimal, :raw_value => '7.6')
-    s.value.should == 7.6
+    s = ActiveSetting::Setting.new(data_type: :decimal, raw_value: '7.6')
+    s.value.should eq 7.6
 
-    s = ActiveSetting::Setting.new(:data_type => :symbol, :raw_value => 'hello')
-    s.value.should == :hello
+    s = ActiveSetting::Setting.new(data_type: :symbol, raw_value: 'hello')
+    s.value.should eq :hello
 
-    s = ActiveSetting::Setting.new(:raw_value => 'hello')
-    s.value.should == 'hello'
+    s = ActiveSetting::Setting.new(raw_value: 'hello')
+    s.value.should eq 'hello'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.